src/main/java/dev/openfeature/sdk/exceptions/FlagNotFoundError.java

```diff
@@ -9,4 +9,9 @@
 public class FlagNotFoundError extends OpenFeatureError {
     private static final long serialVersionUID = 1L;
     @Getter private final ErrorCode errorCode = ErrorCode.FLAG_NOT_FOUND;
+
+    @Override
+    public Throwable fillInStackTrace() {
```

Comment:
```suggestion
    public synchronized Throwable fillInStackTrace() {
```

Sonarcloud is recommending this: https://sonarcloud.io/project/issues?resolved=false&severities=BLOCKER%2CCRITICAL%2CMAJOR%2CMINOR&sinceLeakPeriod=true&types=BUG&pullRequest=739&id=open-feature_java-sdk&open=AYyHw5peA4eBhJGjCSRs

Reply:
oh true, the method in Throwable is sync 🤦